build.gradle

```diff
@@ -23,7 +23,7 @@ plugins {
 
 group = 'io.neonbee'
 // have to use single quotes for XMake, change as soon as new version Gradle build plugin is merged
```

Comment:
shall we remove all references to "XMake" in the comments? Also maybe search for it globally

Reply:
fixed